internal/armada/server/submit_from_log.go

```diff
@@ -121,29 +119,18 @@ func (srv *SubmitFromLog) Run(ctx context.Context) error {
 			lastPublishTime = msg.PublishTime()
 			numReceived++
 
-			// Incoming gRPC requests are annotated with a unique id,
```

Comment:
I've removed this because a) this code is going away soon and b) we've largely broken the link between pulsar message and grpc request so it's not that useful.